modules/installation-creating-gcp-bootstrap.adoc

```diff
@@ -26,12 +26,8 @@ have to contact Red Hat support with your installation logs.
 
 .Prerequisites
 
-* Configure a GCP account.
-* Generate the Ignition config files for your cluster.
-* Create and configure a VPC and associated subnets in GCP.
-* Create and configure networking and load balancers in GCP.
-* Create control plane and compute roles.
-* Ensure pyOpenSSL is installed.
+* Ensure you defined the variables in the _Exporting common variables_ section.
```

Comment:
FYI some variables referenced in this section, `CLUSTER_NETWORK` and `CONTROL_SUBNET` and `ZONE_0`, are defined in the section "Creating load balancers in GCP", rather than in "Exporting common variables" section. So?